pkg/sql/codegen/intermediate_representation.go

```diff
@@ -117,6 +117,8 @@ type AnalyzeIR struct {
 	// "select ... analyze ... with analyze.plot_type = "bar"",
 	// the Attributes will be {"analyze.plot_type": "bar"}
 	Attributes map[string]interface{}
+	// SQLFlow supports TreeExplainer so far.
```

Comment:
This comment is likely to be out of date. Please change to "Explainer types. For example TreeExplainer.".

The list of supported explainers should be documented elsewhere. :)